apps/ui/src/components/inputs/editTextInput.tsx

```diff
@@ -21,7 +21,7 @@ import { TrashIcon } from '../icons/trashIcon';
 interface IEditTextValueInput extends InputProps {
   title: string;
   modalType: string;
-  onSave: (metadata: Metadata) => Promise<void>;
+  onMetadataChange: (metadata: Metadata) => void;
```

Comment:
check this suggestion: https://github.com/infinitybase/bako-id/pull/39#discussion_r1705715461